src/core/database/database.service.ts

```diff
@@ -400,7 +400,7 @@ export class DatabaseService {
         const list = typeof prop === 'object' ? prop.list : false;
 
         if (list) {
-          const value = row[propName] ? row[propName].split(',') : [];
+          const value = row[propName] ? row[propName] : [];
```

Comment:
```suggestion
          const value = row[propName] ?? [];
```

Reply:
Nice! I've never seen this operator before 😆

Reply:
Same as `||` (which I see you used in other lines) it's just stricter.
`||` chooses RHS on falsy values
`??` chooses RHS on null or undefined values